### PR TITLE
docs: used fixed key for keplr set-up

### PR DESCRIPTION
**Pull request by dckc**
refs https://github.com/Agoric/agoric-3-proposals/issues/5

@kbennett2000 I'd like you to test this thoroughly in a bit. The ink is still drying just now...

cc @turadg

### CODE REVIEW

File: main/guides/getting-started/index.md [hunk 1]
```diff
@@ -277,31 +277,10 @@ Once the plug-in has been installed, open Keplr and select the option to **Impor
 <img alt='Choose the "Use recovery phrase or private key" option' src="./assets/040.png"
   width="500" />
 
-To import your wallet, you'll need to copy your mnemonic phrase into Keplr. You can find this series of 24 words back on your terminal window. Copy from this window into your Keplr wallet, then hit the "Import" button.
-
-Please note that your phrase might not be the same as the one shown in this guide!
-
-Note the mnemonic phrase in the output below:
+To import your wallet, you'll need to copy the following mnemonic phrase into Keplr.
```
Comment (turadg) [inline main/guides/getting-started/index.md:280]: consider leaving a code comment about where the key comes from https://github.com/Agoric/agoric-3-proposals/tree/main/packages/synthetic-chain/public/upgrade-test-scripts/keys_for_test_only

**Comment by dckc** [comment]
@kbennett2000 I think this is ready to test.

**Comment by dckc** [comment]
one problem I see:

```
$ yarn docker:bash
yarn run v1.22.21
$ docker compose exec agd bash
WARN[0000] /home/connolly/projects/dapp-agoric-basics/docker-compose.yml: `version` is obsolete 
ENV_SETUP starting
ENV_SETUP finished
========== GOVERNANCE KEYS ==========
bash: /usr/src/upgrade-test-scripts/keys_for_test_only/*.key: No such file or directory
*:
cat: '/usr/src/upgrade-test-scripts/keys_for_test_only/*.key': No such file or directory

========== GOVERNANCE KEYS ==========
```

**Comment by dckc** [comment]
The given mnemonic gives `agoric1rwwley550k9mmk6uq6mm6z4udrg8kyuyvfszjk`
but in the container, `$USER1ADDR` is `agoric1t83za2h5zc5anmkxvfag82qafshr538mvzmnmx`

File: main/guides/getting-started/index.md (hunk 1 above)
Comment (dillu24) [reply main/guides/getting-started/index.md:280]: small note that the mnemonic is wrong. I tried this out on dApp offer up and it seems that user1 pertains to a different mnemonic on-chain.

**Review by kbennett2000 — DISMISSED** [review]

**Comment by dillu24** [comment]
I don't think that the mnemonic added in this commit https://github.com/Agoric/documentation/pull/1048/commits/00a37ce4c4663b7fe9942ba0be39cfbebd121165 is the right one ..

**Comment by dckc** [comment]
we seem to be blocked on...

 - https://github.com/Agoric/agoric-3-proposals/issues/136

**Review by kbennett2000 — APPROVED** [review]
Works as expected

**Pull request merged**